src/helpers.js

```diff
@@ -570,6 +570,9 @@ export function solveUpdate(affectedPaths, comparedPaths) {
  * @return {array}                 - The spliced array.
  */
 export function splice(array, startIndex, nb, ...elements) {
+  if (nb === undefined) {
```

Comment:
Calling `[0, 1, 2, 3, 4].splice(2, null)` does not remove any elements, as does the implementation in my PR.

See the [official splice specification](http://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.splice):

> Let dc be ToInteger(deleteCount).

Where dc will be the eventual delete count. [ToInteger is defined as](http://www.ecma-international.org/ecma-262/6.0/#sec-tointeger)

> 1. Let number be ToNumber(argument).
> 1. ReturnIfAbrupt(number).
> 1. If number is NaN, return +0.
> 1. If number is +0, −0, +∞, or −∞, return number.
> ...

Where [ToNumber is defined](http://www.ecma-international.org/ecma-262/6.0/#sec-tonumber) as returning `+0` when supplying `null` as the argument.

Obviously, when supplying values which are not a number, a parseable string, or null, the implementation will fail.